pom.xml

```diff
@@ -1095,7 +1095,7 @@
           <version>${maven.invoker.plugin.version}</version>
         </plugin>
         <plugin>
-          <groupId>org.jboss.jandex</groupId>
+          <groupId>io.smallrye</groupId>
```

Comment:
Not sure if this might cause issues with Quarkus. We need to verify that the Jandex versions are compatible.

Reply:
I'm not sure, maybe use jandex directly instead of the plugin? I think that would require adding empty beans.xml files, though…

Reply:
I just checked and Quarkus is using this Jandex distribution and version, so we should be fine:

- https://github.com/quarkusio/quarkus/blob/e3c7965ca331d8761897c52b56d5b4aa2c0fa114/bom/application/pom.xml#L23
- https://github.com/quarkusio/quarkus/blob/e3c7965ca331d8761897c52b56d5b4aa2c0fa114/bom/application/pom.xml#L4480-L4490
- https://github.com/quarkusio/quarkus/blob/dd0c2410286b82ec111be5e87f03024caff081a2/build-parent/pom.xml#L561-L565